pom.xml

```diff
@@ -20,7 +20,7 @@
 		<java.version>1.8</java.version>
 		<kubernetes-client.version>1.3.83</kubernetes-client.version>
 		<kubernetes-assertions.version>2.2.110</kubernetes-assertions.version>
-		<spring-cloud-deployer-spi.version>1.0.1.RELEASE</spring-cloud-deployer-spi.version>
+		<spring-cloud-deployer-spi.version>1.0.2.BUILD-SNAPSHOT</spring-cloud-deployer-spi.version>
```

Comment:
yeah, it is `1.0.2`. The commit description for the release had it as `1.0.3.RELEASE` in `spring-cloud-deployer` which confused me.